compute-budget-instruction/src/instructions_processor.rs

```diff
@@ -279,6 +279,14 @@ mod tests {
             Err(TransactionError::DuplicateInstruction(2))
         );
 
+        test!(
```

Comment:
not oppposed to a new test, but why was this added as part of this PR?

Reply:
This was to accommodate for the test changes in transaction_processor.rs due to removal of usage of `process_compute_budget_instructions()`.